development-updates.md

```diff
@@ -8,31 +8,32 @@ Add your name **alphabetically** and link your updates to the corresponding tabl
+| [DanGoron](https://github.com/gorondan)                      | [Update 0](https://hackmd.io/@meA4-YJOSaqtagggpgcWMg/SJmdOEmXR)                                                                                  |                                                                 |        |     |     |     |
+| [Ekaterina Riazantseva](https://github.com/KatyaRyazantseva) | [Update 0](https://hackmd.io/@sWdTSBHnRb63bJ_wVjNsyA/Hk8wjVwSC)                                                                                  |                                                                 |        |     |     |     |
+| [georgesheth](https://github.com/georgesheth)                | [Update 0](https://hackmd.io/@georgesheth/SJ2FqiVSR)                                                                                             |                                                                 |        |     |     |     |
+| [ghili](https://github.com/ghiliweld)                        | [Update 0](https://hackmd.io/@ghili/HJoy-VBS0)                                                                                                   | [Update 1](https://hackmd.io/@ghili/ry9-_kISR)                  |        |     |     |     |
+| [Hamid Bateni](https://github.com/irnb)                      | [Update 0](https://hackmd.io/@irnb/epf-update-0)                                                                                                 |                                                                 |        |     |     |     |
+| [Hangleang](https://github.com/hangleang)                    | [Update 0](https://hackmd.io/@hangleang/epf-week-0)                                                                                              | [Update 1](https://hackmd.io/@hangleang/epf5-week1)             |        |     |     |     |
+| [Hopinheimer](https://github.com/hopinheimer)                | [Update 0](https://hackmd.io/@np8VhkKRRHanpT1vbtRQ4Q/ByAoT58Q0)                                                                                  |                                                                 |        |     |     |     |
+| [Krishang](https://github.com/kamuik16)                      |                                                                                                                                                  | [Update 1](https://hackmd.io/@kamuik16/SyVy4aKrR)               |        |     |     |     |
+| [Ktl_XV](https://github.com/ktl-xv)                          |                                                                                                                                                  | [Update 1](https://hackmd.io/@Ktl-XV/EPF5-W1)                   |        |     |     |     |
```

Comment:
This is my actual change

Reply:
It's because the formatter you used created 3 extra columns. Please remove those first

Reply:
:facepalm: Fixed

Reply:
No problem, thanks for the update! Love the whiteboard brainstorming :)